### DISCUSSION TITLE
I think consistent-type-imports should be added to recommended config.

### DISCUSSION THREAD

**Discussion by lovetingyuan** [typescript-eslint/typescript-eslint#7629] [Community Feedback]
I think [consistent-type-imports](https://typescript-eslint.io/rules/consistent-type-imports/) should be added to recommended config.
This has benefits for dependency analysis tools, build tools, and code maintainability. I find these points quite valuable.

**Comment by zhangciwu** [comment]
https://github.com/typescript-eslint/typescript-eslint/issues/4268
Perhaps it conflicts with the TSC in some cases, as mentioned above.
